templates/unstable-vite-cloudflare/load-context.ts

```diff
@@ -0,0 +1,34 @@
+import type { KVNamespace } from "@cloudflare/workers-types";
+import { type AppLoadContext } from "@remix-run/cloudflare";
+
+// In the future, types for bindings will be generated by `wrangler types`
+// See https://github.com/cloudflare/workers-sdk/pull/4931
+type Bindings = {
+  // Add types for bindings configured in `wrangler.toml`
+  MY_KV: KVNamespace;
+};
+
+declare module "@remix-run/cloudflare" {
+  interface AppLoadContext {
+    env: Bindings;
+
+    extra: string; // Example of augmenting load context
```

Comment:
I think this line and the implementation should be commented out so that the load context doesn't have extra stuff (heh) on it.

Alternatively, do you think this requirement is rare enough that it could just be documented rather than having it in the template? I'm happy either way, I just want to make sure we're not clogging up the template with an edge case.